lib/srv/ctx.go

```diff
@@ -158,6 +158,10 @@ type IdentityContext struct {
 
 	// RouteToCluster is derived from the certificate
 	RouteToCluster string
+
+	// ActiveRequests is a string of active access request ID's. Example layout:
+	// {"access_requests":["xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx","xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"]}
+	ActiveRequests string
```

Comment:
Nit: Probably more useful if we store the parsed form (i.e. `[]string`) here.  That way if we add more logic down the road that uses the active requests field, we don't have to parse in multiple places.

Reply:
Moved the logic.